app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,15 @@
+class CommentsController < ApplicationController
+  def new; end
+
+  def create
+    data = params.require(:comment).permit(:text)
+    post = Post.find(params[:post_id])
+    comment = Comment.create!(
+      post: post,
+      author: current_user,
+      text: data[:text]
+    )
+    comment.update_comments_counter_for_post
+    redirect_to user_posts_path
+  end
```

Comment:
Optional -
it will be a nice addition if you added  a nice 
 [params method](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md#:~:text=Pay%20special%20attention%20to%20the%20strong%20parameters%20description%20in%20Rails%20Guides) 
that will look somewhat like this  
`
    private 
    def post_params
      params.require(:post).permit(:title, :text)
    end
`
😄